python/src/viewer.py

```diff
@@ -94,11 +93,32 @@ class View(object):
         Adds a legend if True. Default is True.
     square_axes : bool, optional
         Forces the axes to share the same scale if True. Default is False.
+        
+    Examples
+    --------
+    >>> import openturns as ot
+    >>> from openturns.viewer import View
+    >>> graph = ot.Normal().drawPDF()
+    >>> view = View(graph, plot_kwargs={'color':'blue'})
+    >>> view.save('graph.png', dpi=100)
+    >>> view.show()
     """
 
     @staticmethod
     def CheckDict(arg):
-        """Check that the argument is a python dictionary."""
+        """
+        Check that the argument is a Python dictionary.
+        
+        Examples
+        --------
+        >>> import openturns as ot
+        >>> import openturns.viewer as otv
+        >>> plot_kwargs = {'color':'blue'}
+        >>> plot_kwargs = otv.View.CheckDict(plot_kwargs)
```

Comment:
I think example should stops here

Reply:
This is not clear: what example would you add or remove? Why?

Reply:
Do you mean that the `CheckDict` method should not be documented?

Reply:
it's just that it's not meant to be used from outside openturns

Reply:
yes but this is a static method inside the class
I will move it latter outside the `View` class

Reply:
yes

Reply:
Fixed at ecd24ab80b0bb4a3fc90341020514e6533620bcb. Also renamed `ToUnicode` into `_ToUnicode` for the same reason.